patches/minecraft/net/minecraft/tags/TagLoader.java.patch

```diff
@@ -29,15 +29,6 @@
     }
  
     public Map<ResourceLocation, Collection<T>> m_203898_(Map<ResourceLocation, List<TagLoader.EntryWithSource>> p_203899_) {
-@@ -100,7 +_,7 @@
-       });
-       dependencysorter.m_284430_((p_284682_, p_284683_) -> {
-          this.m_215978_(lookup, p_284683_.f_283922_).ifLeft((p_215977_) -> {
--            f_13445_.error("Couldn't load tag {} as it is missing following references: {}", p_284682_, p_215977_.stream().map(Objects::toString).collect(Collectors.joining(", ")));
-+               f_13445_.error("Couldn't load tag {} as it is missing following references: {}", p_215977_, p_215977_.stream().map(Objects::toString).collect(Collectors.joining(", \n\t")));
```

Comment:
The actual issue here seems to stem from a patching error from the 1.19 update: https://github.com/neoforged/NeoForge/commit/4ed9117799b44611fe978c0de8f5c483df600191#diff-25fbb9cdc0f246b5c8f16799b4cbb56a374f5c14edaecc7decc6d959d8dc5766.

Specifically, the two arguments passed to the logging call should be different (as the removal line indicates), but was accidentally changed to being the same argument twice.

Removing the patch _is_ a solution, but it also removes the added `\n\t` at the end of the line, which was included to make looking at tag errors easier--each missing reference being on a separate line, rather than all bunched on the same line.

I'd heavily prefer keeping the `\n\t` patch as-is, and fixing the first argument to the logging call to the proper one (as seen in the removal).